dashboard/pages/primary_screening/callbacks.py

```diff
@@ -0,0 +1,116 @@
+from dash import html, no_update, callback, Output, Input, State
+
+from ..error import throwable, DashboardError
+from ...data.bmg_plate import parse_bmg_files_from_iostring
+from ...storage import FileStorage
+from ...data.file_preprocessing.echo_files_parser import EchoFilesParser
+import io
+import base64
+import functools
+import uuid
+
+# ==== upload_stage callbacks ====
+def validate_content(contents, filenames):
+    if any(filename[-4:] != "txt" for filename in filenames):
```

Comment:
Probably meant ".txt"
Btw, is this function used? I see it being commented

Reply:
For now it is no needed so I deleted this